public/js/clientLIst.js

```diff
@@ -0,0 +1,71 @@
+const tableContent = document.getElementById('clients-table-content');
+
+const openEditClient = (id) => {
+  window.location.href = `${window.location.origin}/views/clientForm.html?_id=${id}`;
+};
+
+const openNewClient = () => {
+  window.location.href = `${window.location.origin}/views/clientForm.html`;
+};
+
+const deleteClient = (id, name, event) => {
+  event.stopPropagation();
+  document.getElementById('modal-title').innerText = 'You are about to delete';
+  document.getElementById('modal-data-inputs').innerText = `${name}`;
```

Comment:
no need to use string template here
```suggestion
  document.getElementById('modal-data-inputs').innerText = name;
```

Reply:
done!